MTFO/Patches/Patch_DiscordManager.cs

```diff
@@ -0,0 +1,23 @@
+using MTFO.Utilities;
+using HarmonyLib;
+
+namespace MTFO.Patches
+{
+    [HarmonyPatch(typeof(DiscordManager), nameof(DiscordManager.UpdateDiscordDetails))]
+    static class Patch_DiscordManager
+    {
+        [HarmonyPrefix]
+        private static bool Pre_UpdateDiscordDetails(eDiscordDetailsDisplay details)
+        {
+            if (details == null) return false;
+            if (Globals.Global.RundownIdToLoad != 1) return true;
```

Comment:
Is this check sane with the multiple rundowns addition they made? How has this been tested?

Reply:
If details cannot be null (enum), the line is unnecessary. I just want/need to null-check the only variable in play, one way or another :)

This has been compiled and -tested with modded rundown id 1 with completeobjective 0 (OnSuccess) and suicide (OnFail) separately. Tests performed without LocalProgression or FixEndScreen (no problem when either is present)

I have played MTFO-enabled vanilla rundown expeditions with original rundown ids and without the mentioned mods extensively and never had any problems on success/fail screen.

Reply:
Why are you still checking if the enum is null? This isnt Java. enums cannot be null! Any comparison of a value type to null automatically gets rid of said comparison when it gets compiled because value types (like enums) cannot be null. See [this](https://sharplab.io/#v2:EYLgtghglgdgNAExAagD4AEBMBGAsAKAIBEIAXAUwBUoxyACAZzoF46SLraA6AOQHsA7gG4CBKADM6ACibNWMAK4AbJQEoCdOumwBOKQCJSACyhNTMAOSk6sAMZKFCcgn2qR+ckobkNW3QcoTMzMYe0dnV3cCbT19AGU+GyZjU0igA==)

![Screenshot_20240602_124709_Brave.jpg](https://github.com/GTFO-Modding/MTFO/assets/29237603/7cda25a1-a8bb-4325-a1b7-cc25bcdde863)

This null check is completely useless and thus should be removed